specs/pvs/corla.pvs

```diff
@@ -863,6 +934,8 @@ BEGIN
 % @description The audited contest determines the number of ballots
 % that must be examined and verified during the RLA.
```

Comment:
At the moment the information from CDOS is sufficiently underspecified that we have not presume that fractional risk limits are permitted and even desired.  We could, of course, encode non-transcendental risk limits as natural numbers using a mantissa, by why bother?